clang/lib/CodeGen/Targets/AArch64.cpp

```diff
@@ -1143,30 +1146,62 @@ void AArch64TargetCodeGenInfo::checkFunctionABI(
   }
 }
 
-void AArch64TargetCodeGenInfo::checkFunctionCallABIStreaming(
-    CodeGenModule &CGM, SourceLocation CallLoc, const FunctionDecl *Caller,
-    const FunctionDecl *Callee) const {
-  if (!Caller || !Callee || !Callee->hasAttr<AlwaysInlineAttr>())
-    return;
+enum class ArmStreamingInlinability : uint8_t {
```

Comment:
nit: This is a bit of a misnomer, because NewZA has nothing to do with streaming mode. What about calling it `ArmInlineCompatibility` ?

Reply:
Gone with `ArmSMEInlinability` (since this is only about SME ABI inlinability issues).